app/src/controllers/pages/selectors.js

```diff
@@ -210,24 +210,26 @@ export const userAssignedSelector = (projectSlug, organizationSlug) => (state) =
   const isAdmin = userRole === ADMINISTRATOR;
   const isManager = organizationRole === MANAGER;
   let isAssignedToTargetOrganization = false;
+  const projectKey = `${organizationSlug}.${projectSlug}`;
```

Comment:
I'm not sure the `projectKey` will be always look like this.
Can we rely on slugs here?

Reply:
The slugs can be the same for projects. The uniqueness of the projects is ensured only by the project key.
This is done for backward compatibility. But then helen.helen is expected
![image](https://github.com/user-attachments/assets/3f4ba7d7-2d28-466d-ad07-ef227c295fdc)

Reply:
Done.